frontend/simple/utils/distribution/mincome-default.js

```diff
@@ -1,9 +1,6 @@
 'use strict'
 
-type IncomeObject = {
-  name: string,
-  amount: number
-}
+import type {IncomeObject} from '../../../../shared/types'
```

Comment:
All imports must specify the extension. So here and the other places add a `.js`

Reply:
👍 fixed